### PR TITLE
s390x: reselect svirt console to extract SUT pty

### PR DESCRIPTION
**Pull request by Soulofdestiny**
verification run: http://opeth/tests/70
@rwx788 @nicksinger 

@mnowaksuse maybe also interesting for you - though it should not influence other svirt tests

### CODE REVIEW

File: lib/utils.pm [hunk 1]
```diff
@@ -114,6 +114,9 @@ sub unlock_if_encrypted {
     return unless get_var("ENCRYPT");
 
     if (check_var('ARCH', 's390x') && check_var('BACKEND', 'svirt')) {
+        # we need to reset the svirt console to extract the pty variable correctly
+        console('svirt')->reset;
+        select_console('svirt');
```
Comment (okurz) [inline lib/utils.pm]: IIUC the wrong location. It needs to be set whenever we reboot machines not here

Comment (Soulofdestiny) [reply lib/utils.pm]: yeah - also discussed with @rwx788, makes sense to move to `redefine_svirt_domain`

**Comment by Soulofdestiny** [comment]
changed - new verification run: http://opeth/tests/70
(also updated in initial description)

**Comment by Soulofdestiny** [comment]
new verification run passed

**Comment by okurz** [comment]
why is http://opeth/tests/70#step/boot_encrypt/2 still red then?

**Comment by Soulofdestiny** [comment]
that's indeed a good question - it timed out... but passphrase was typed...

**Comment by Soulofdestiny** [comment]
not needed anymore, was fixed in the meantime

**Pull request closed without merging**